content/src/view.js

```diff
@@ -1324,7 +1324,44 @@ $(window).on('resize.listing', function() {
   }
 });
 
-function updatePaneLinks(pane) {
+
+var searchCache={};
```

Comment:
Consider putting the search cache object definition in a separate searchcache.js file.